### PR TITLE
Add "Important" to the managed devices callout.

### PR DESCRIPTION
**Pull request by bpeterse03**
Would like to add an "Important" notification to the managed device callout.

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/knowledge/set-up-topic-experiences.md [hunk 1]
```diff
@@ -20,6 +20,8 @@ It is important to plan the best way to set up and configure topics in your envi
 
 You must be [subscribed to Viva Topics](https://www.microsoft.com/microsoft-viva/topics) and be a global administrator or SharePoint administrator to access the Microsoft 365 admin center and set up Topics.
 
+## Important ##
```
Comment (JohanFreelancer9) [inline microsoft-365/knowledge/set-up-topic-experiences.md]:
```suggestion
> [!IMPORTANT]
```

File: microsoft-365/knowledge/set-up-topic-experiences.md [hunk 2]
```diff
@@ -20,6 +20,8 @@ It is important to plan the best way to set up and configure topics in your envi
 
 You must be [subscribed to Viva Topics](https://www.microsoft.com/microsoft-viva/topics) and be a global administrator or SharePoint administrator to access the Microsoft 365 admin center and set up Topics.
 
+## Important ## 
+
```
Comment (JohanFreelancer9) [inline microsoft-365/knowledge/set-up-topic-experiences.md]:
```suggestion
```

File: microsoft-365/knowledge/set-up-topic-experiences.md [hunk 3]
```diff
@@ -20,6 +20,8 @@ It is important to plan the best way to set up and configure topics in your envi
...
 If you have configured SharePoint to [require managed devices](https://docs.microsoft.com/sharepoint/control-access-from-unmanaged-devices), be sure to set up Topics from a managed device.
```
Comment (JohanFreelancer9) [inline microsoft-365/knowledge/set-up-topic-experiences.md]:
```suggestion
> If you have configured SharePoint to [require managed devices](https://docs.microsoft.com/sharepoint/control-access-from-unmanaged-devices), you must set up Topics from a managed device.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@petey121175

**Comment by yogkumgit** [comment]
Reminder @petey121175

**Review by Josegto1 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@JohanFreelancer9

**Comment by Josegto1** [comment]
AP

**Review by Josegto1 — APPROVED** [review]
GJ

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**